submissions/ses3332002/memory-game/script.js

```diff
@@ -0,0 +1,140 @@
+"use strict";
+  
+  const doc = document.body;
+  const themeButtons = document.querySelector(".theme_selector");
+  const sizeSelector = document.querySelector(".size_selector");
+  const cardThemeSelector = document.querySelectorAll('input[name="option"]');
+  const startButton = document.querySelector(".start_button");
+  const gameArea = document.querySelector(".game_area");
+  const switchableControls = document.querySelectorAll(".switchable");
+  let numbersArray;
+  let cardThemeSelected;
+  let cardSelected;
+  let cardsArray;
+  let matchedCounter;
+  
+  startButton.addEventListener("click", startGame);
+  doc.classList.add("theme1");
+  
+  themeButtons.addEventListener("click", function({target}) {
+    if (target.tagName != "BUTTON") {
+      return;
+    };
+    doc.classList.remove(doc.classList[0]);
+    doc.classList.add(target.dataset.theme);
+  });
+  
+  function startGame() {
+    cardThemeSelected = document.querySelector('input[name="option"]:checked');
```

Comment:
Why do you not declare this variable as a constant at the beginning?

Reply:
Player can switch theme of cards many times, so this is variable, not constant.